services/resources-getter.js

```diff
@@ -283,7 +287,11 @@ function ResourcesGetter(model, opts, params) {
     if (params.search) {
       _.each(schema.fields, function (field) {
         if (field.search) {
-          field.search(countOpts, params.search);
+          try {
+            field.search(countOpts, params.search);
+          } catch (error) {
+            console.log(error);
+          }
```

Comment:
You duplicate the code doing this (the current code is already duplicated...), I think it is time for code factorisation.